src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -202,67 +213,71 @@
+                    <DataCollection
+                      v-slot="{ items }"
+                      :predicate="route.params.inactive ? undefined : (item) => ((item.protocol === 'tcp' ? item.tcp?.downstream_cx_rx_bytes_total : item.http?.downstream_rq_total) as (number | undefined) ?? 0) > 0"
+                      :items="traffic.outbounds"
+                    >
+                      <ServiceTrafficGroup
+                        v-if="items.length > 0"
+                        type="outbound"
+                        data-testid="dataplane-outbounds"
+                      >
+                        <template
+                          v-for="item in items"
+                          :key="`${item.name}`"
                         >
-                          <RouterLink
-                            :to="{
-                              name: 'data-plane-outbound-summary-overview-view',
-                              params: {
-                                service: item.name,
-                              },
-                              query: {
-                                inactive: route.params.inactive ? null : undefined,
-                              },
-                            }"
+                          <ServiceTrafficCard
+                            :protocol="item.protocol"
+                            :traffic="item"
                           >
-                            {{ item.name }}
-                          </RouterLink>
-                        </ServiceTrafficCard>
-                      </template>
-                    </ServiceTrafficGroup>
+                            <RouterLink
+                              :to="{
+                                name: 'data-plane-outbound-summary-overview-view',
+                                params: {
+                                  service: item.name,
+                                },
+                                query: {
+                                  inactive: route.params.inactive ? null : undefined,
+                                },
+                              }"
+                            >
+                              {{ item.name }}
+                            </RouterLink>
+                          </ServiceTrafficCard>
+                        </template>
+                      </ServiceTrafficGroup>
+                    </DataCollection>
                   </template>
                 </template>
+                <template v-else>
+                  <EmptyBlock />
+                </template>
               </DataPlaneTraffic>
             </div>
           </KCard>
           <RouterView
-            v-if="route.params.service && [
-              traffic?.outbounds,
-              props.data.dataplane.networking.inbounds,
-            ].every(item => typeof item !== 'undefined')"
             v-slot="child"
           >
             <SummaryView
+              v-if="child.route.name !== route.name"
```

Comment:
**Question**: This doesn’t look right. Why are we no longer guarding this by the specific route parameter (i.e. `route.params.service`) we know we need to guard against?

Reply:
If child route is ever not the name of this route i.e. `"data-plane-detail-view"` then it means we must be in a "sub view" i.e. in this case a summary view seeing as "data-plane-detail-view"` has no other types of sub view, therefore show the SummaryView. That makes sense I think?

Reply:
That is coincidentally true as long as don’t have other child routes in this view that aren’t summary trays. The check for `route.params.service` is more robust. A check for the intended sub routes would be even more robust.

Reply:
There're no other child routes in this route that aren't a SummaryViews are there? Whilst I wouldn't completely rule it out I'm 90% sure we'd probably never have a subroute here that isn't a SummaryView and if that ever did happen we would have to change quite a bit more I think.

Reply:
Feels like I can resolve this one?